### PR TITLE
Scala @deprecated now generates javadoc @deprecated

### LINKED ISSUE #60: deprecated annotation not propagated to javadoc

### PR DESCRIPTION
**Pull request by ktoso**
Fixes #60 

Suboptimal list concatenations here and there, should I play around with `reverse` tricks?

### CODE REVIEW

**Comment by ktoso** [comment]
Please review once you have a sec @rkuhn :)

**Comment by ktoso** [comment]
Oh seems I didn't run all tests somehow, will fix

**Comment by ktoso** [comment]
Don't quite understand why it fails like this on travis... will need to look into it later.

**Comment by patriknw** [comment]
LGTM

File: tests/src/test/scala/scala/javadoc/test.scala [hunk 1]
```diff
@@ -225,6 +225,31 @@ private[it] object PPrivate {
...
+  /**
+   * Some methods are forever.
+   */
+  @deprecated("This is replaced by someDiamondsAre", since = "now")
+  def orNotSoMuch = ()
+
+  /**
+   * @deprecated This is already deprecated. Since now.
+   */
+  @deprecated("This is already deprecated", since = "now")
```
Comment (rkuhn) [inline tests/src/test/scala/scala/javadoc/test.scala]: Please use a different message so that it is clear which one takes precedence.

**Comment by rkuhn** [comment]
LGTM

File: tests/src/test/scala/scala/javadoc/test.scala (hunk 1 above)
Comment (ktoso) [inline tests/src/test/scala/scala/javadoc/test.scala]: okey, will do

**Comment by ktoso** [comment]
PS validation passed, on all versions of scala - yay!

**Pull request merged**